charts/clusterplex/templates/pms.yaml

```diff
@@ -0,0 +1,165 @@
+{{- define "clusterplex.pms.hardcodedValues" -}}
+global:
+  nameOverride: "pms"
+
+image:
+  repository: '{{ .Values.global.plexImage.repository }}'
+  tag: '{{ .Values.global.plexImage.tag | default "latest" }}'
+  imagePullPolicy: '{{ .Values.global.plexImage.imagePullPolicy }}'
+
+configMaps:
+  config:
+    enabled: true
+    data:
+      VERSION: '{{ .Values.pms.config.version | default "docker" }}'
+      TZ: '{{ .Values.global.timezone | default "America/Chicago" }}'
+      PGID: '{{ .Values.global.PGID | default "1000" }}'
+      PUID: '{{ .Values.global.PUID | default "1000" }}'
+      DOCKER_MODS: 'ghcr.io/pabloromeo/clusterplex_dockermod:{{ .Values.global.clusterplexVersion | default .Chart.AppVersion }}'
+      ORCHESTRATOR_URL: 'http://{{ .Release.Name }}-orchestrator:{{ .Values.orchestrator.config.port | default "3500" }}'
+      PMS_SERVICE: '{{ .Release.Name }}-pms'
+      PMS_PORT: '{{ .Values.pms.config.port | default "32400" }}'
+      TRANSCODER_VERBOSE: '{{ .Values.pms.config.transcoderVerbose | default "1" }}'
+      TRANSCODE_OPERATING_MODE: '{{ .Values.pms.config.transcodeOperatingMode | default "both" }}'
+      LOCAL_RELAY_ENABLED: '1'
```

Comment:
Any reason why disabling the Relay is not available through values.yaml? Just curious

Reply:
Let me know if you want me to merge as-is and we add that additional parameter at a later time, or if you want me to wait for an update. Don't think many will want to be turning that off anyways.

Reply:
I originally did not add that in to keep it simple for the initial version. If you want to merge it in I can create a new PR adding the ability to change that in another version.

Reply:
Sure thing, let's leave it as is.